cpusupport/cpusupport_arm_crc32_64.c

```diff
@@ -0,0 +1,23 @@
+#include "cpusupport.h"
+
+#ifdef CPUSUPPORT_HWCAP_GETAUXVAL
+#include <sys/auxv.h>
+#endif /* CPUSUPPORT_HWCAP_GETAUXVAL */
+
+CPUSUPPORT_FEATURE_DECL(arm, crc32_64)
+{
+	int supported = 0;
+#if defined(CPUSUPPORT_ARM_CRC32_64)
+#if defined(CPUSUPPORT_HWCAP_GETAUXVAL)
+	unsigned long capabilities;
+
+#if defined(__aarch64__)
+	capabilities = getauxval(AT_HWCAP);
+	supported = capabilities & HWCAP_CRC32;
+#endif
+#endif /* CPUSUPPORT_HWCAP_GETAUXVAL */
+#endif /* CPUSUPPORT_ARM_CRC32_64 */
+
+	/* Return the supported status. */
+	return (supported);
```

Comment:
Didn't we decide that these functions should return 1 or 0 rather than a particular bit out of a mask?

Reply:
Right, fixed with `(supported > 0)`.  I considered using `(supported ? 1 : 0)` which is the closest to what all the other "runtime detection" files do, but that seemed unnecessarily complicated.